src/main/java/edu/cornell/kfs/concur/batch/service/ConcurRequestedCashAdvanceService.java

```diff
@@ -1,11 +1,15 @@
 package edu.cornell.kfs.concur.batch.service;
 
+import java.util.Collection;
+
 import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestedCashAdvance;
 
 public interface ConcurRequestedCashAdvanceService {
     
     void saveConcurRequestedCashAdvance(ConcurRequestedCashAdvance concurRequestedCashAdvance);
     
     boolean isDuplicateConcurRequestCashAdvance(ConcurRequestedCashAdvance concurRequestedCashAdvance);
-    
+
+    Collection<ConcurRequestedCashAdvance> findConcurRequestedCashAdvanceBy(String cashAdvanceKey);
```

Comment:
It might be a good idea to have the method name reflect what type of ID/key is being searched on, rather than only having it in the argument name.